sources/packages/api/src/route-controllers/note/models/note.dto.ts

```diff
@@ -1,15 +1,19 @@
+import { IsNotEmpty } from "class-validator";
+
 /**
  * Base DTO for note.
  */
-export interface NoteBaseDTO {
+export class NoteBaseAPIInDTO {
+  @IsNotEmpty()
```

Comment:
Note type must be specific about the possible types to be accepted.
If the entire enum is possible we can use `@IsEnum`, if only a few items of the enum are possible we should use the `@IsIn`.